src/OrchardCore.Modules/OrchardCore.Queries/Views/_ViewImports.cshtml

```diff
@@ -11,4 +11,7 @@
 @using Microsoft.Extensions.Localization;
 @using Microsoft.AspNetCore.Mvc.Localization;
 @using OrchardCore.ContentManagement.Display
+@using OrchardCore.DisplayManagement.Views
 @using OrchardCore.Queries.ViewModels
+@using OrchardCore.Queries.Sql.Deployment
+@using OrchardCore.Queries.Sql.ViewModels
```

Comment:
Why are these changes needed?

Reply:
I did it because I wanted to avoid repeating those lines inside a few **cshtml** files.
But once you've asked I realized that those are only 3 files, so I moved those usings.